### PR TITLE
Global ROM for linear elasticity

### PR DESCRIPTION
**Pull request by larsson4**

### CODE REVIEW

File: test/test_workflow.cpp [hunk 1]
```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
    return;
 }
 
+TEST(LinElast_Workflow, MFEMIndividualTest)
+{
+   config = InputParser("inputs/linelast.base.yml");
```
Comment (dreamer2368) [inline test/test_workflow.cpp:559]: You'll have to include `inputs/linelast.base.yml` in this PR

File: test/test_workflow.cpp [hunk 2]
```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
...
+
+   config.dict_["model_reduction"]["rom_handler_type"] = "mfem";
+   config.dict_["model_reduction"]["visualization"]["enabled"] = true;
```
Comment (dreamer2368) [inline test/test_workflow.cpp]: This can be turned off for ci workflow

File: test/test_workflow.cpp [hunk 3]
```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
...
+   config.dict_["model_reduction"]["visualization"]["prefix"] = "basis_paraview";
+   config.dict_["sample_generation"]["parameters"][0]["sample_size"] = 2;
```
Comment (dreamer2368) [inline test/test_workflow.cpp]: We don't really make so many samples here, but can we increase the sample size a bit, say 4?

File: test/test_workflow.cpp [hunk 4]
```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
...
+{
+   config = InputParser("inputs/linelast.base.yml");
+   config.dict_["visualization"]["enabled"] = true;
+
+   config.dict_["model_reduction"]["rom_handler_type"] = "mfem";
+   config.dict_["model_reduction"]["visualization"]["enabled"] = true;
+   config.dict_["model_reduction"]["visualization"]["prefix"] = "basis_paraview";
+
+   config.dict_["single_run"]["linelast_disp"]["rdisp_f"] = 1.0;
+   config.dict_["sample_generation"]["parameters"][0]["sample_size"] = 1;
```
Comment (dreamer2368) [inline test/test_workflow.cpp]: Same as above, maybe increase to 2?

**Review by dreamer2368 — CHANGES_REQUESTED** [review]

File: test/test_workflow.cpp (hunk 1 above)
Comment (larsson4) [reply test/test_workflow.cpp:559]: Oops! Coming right up.

File: test/test_workflow.cpp (hunk 2 above)
Comment (larsson4) [reply test/test_workflow.cpp]: The visualization?

File: test/test_workflow.cpp (hunk 4 above)
Comment (larsson4) [reply test/test_workflow.cpp]: We don't pass the test if we do this

File: test/test_workflow.cpp [hunk 5]
```diff
@@ -553,6 +554,69 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
...
+   printf("Error: %.15E\n", error);
+   EXPECT_TRUE(error < linelast_threshold);
+
+   return;
+}
+
+TEST(LinElast_Workflow, MFEMUniversalTest)
+{
+   config = InputParser("inputs/linelast.base.yml");
+   config.dict_["visualization"]["enabled"] = true;
```
Comment (dreamer2368) [inline test/test_workflow.cpp]: turn off visualization for ci workflow

**Review by dreamer2368 — APPROVED** [review]

**Pull request merged**